backend/app/api/endpoints/impersonation.py

```
@@ -0,0 +1,272 @@
+    return ImpersonationStartResponse(
+        access_token=access_token,
+        token_type="bearer",
+        impersonated_user_id=target_user.id,
+        impersonated_user_name=target_user.user_name,
+        session_expires_at=request.session_expires_at,
+    )
+
+
+@router.post("/impersonate/exit", response_model=ImpersonationExitResponse)
+async def exit_impersonation_session(
+    db: Session = Depends(get_db),
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)